src/deprecated.jl

```diff
@@ -22,3 +22,6 @@ import Base: keys, values, insert!
 
 @deprecate stackdf stackdt
 @deprecate meltdf meltdt
+
+@deprecate readtable CSV.read
```

Comment:
More or less, because people may have expectations when coming from DataFrames, but we don't need to care. :-)